apps/viewer/src/app.tsx

```diff
@@ -39,7 +39,7 @@ const App = () => {
 
   const viewerPlaybackControl = usePlaybackControl(Array.from(remoteTrackSources).map(([sourceId]) => sourceId));
 
-  const [mainSourceId, setMainSourceId] = useState<string>();
+  const [mainSourceId, setMainSourceId] = useState<string | null>();
```

Comment:
Could we replace all `null` with `undefeined` to keep consistency?

Reply:
We need both `null` and `undefined` in order to distinguish between:
- State has not been set yet: we use `undefined`
- State has been set, the main source does not have a label/ID: we use `null`

Reply:
That's confused, let's use `''`(empty string) for empty id.